### ISSUE TITLE
Cann't wait to test it.

### ISSUE THREAD

**Issue by KonVei82** [Pacerino/CaddyProxyManager#2]
As a feature request: SSL Certificate status for each individual proxy host.

**Comment by Gjergj** [comment]
Hey I've been working on a GUI for Caddy https://github.com/Gjergj/proxy_gui 
Currently it's MAC only but I plan to bring it to windows and linux.
It's still early but supports fileserver and proxy server configuration.
Please do create an issue on github about anything, some feedback would help tremendously

**Comment by gedw99** [comment]
> Hey I've been working on a GUI for Caddy https://github.com/Gjergj/proxy_gui Currently it's MAC only but I plan to bring it to windows and linux. It's still early but supports fileserver and proxy server configuration. Please do create an issue on github about anything, some feedback would help tremendously

I guess it's closed source ? I wanted to work on the Windows version

**Comment by Gjergj** [comment]
> > Hey I've been working on a GUI for Caddy https://github.com/Gjergj/proxy_gui Currently it's MAC only but I plan to bring it to windows and linux. It's still early but supports fileserver and proxy server configuration. Please do create an issue on github about anything, some feedback would help tremendously
> 
> I guess it's closed source ? I wanted to work on the Windows version

Yes, it's closed source. I plan to have a windows version in the next two weeks.

I'll drop a message here when it's ready.

**Comment by gedw99** [comment]
> > > Hey I've been working on a GUI for Caddy https://github.com/Gjergj/proxy_gui Currently it's MAC only but I plan to bring it to windows and linux. It's still early but supports fileserver and proxy server configuration. Please do create an issue on github about anything, some feedback would help tremendously
> > 
> > 
> > I guess it's closed source ? I wanted to work on the Windows version
> 
> Yes, it's closed source. I plan to have a windows version in the next two weeks.
> 
> I'll drop a message here when it's ready.

Ok thank you

**Comment by KonVei82** [comment]
Thanx, will wait for your Windows version release.

**Comment by Gjergj** [comment]
Hey @gedw99 @KonVei82 just released a windows version, feel free to create an issue should you need any help

https://github.com/Gjergj/proxy_gui

**Comment by Pacerino** [comment]
Hi there!

I'm really sorry about how things went. A lot has happened behind closed doors in recent years. I've finally got the time and the resources to start this project again, and I've released a new version.

If you could take a look at https://github.com/Pacerino/CaddyProxyManager/releases/tag/v3.1.0-rc1 and leave a review, share any improvements or report any bugs, that'd be great! I really appreciate it!
